### DISCUSSION TITLE
FANSI without GPU usage

### DISCUSSION THREAD

**Discussion by loneblues** [kschan0214/sepia#52] [Q&A] [answered]
Can FANSI or NDI be excuted without GPU? It appears that my GPU does not have enough memory for FANSI, so if there is a way to run FANSI on CPU would be preferred. 
Thanks in advance.

**Comment by kschan0214** [comment, marked as answer]
Hi, unfortunately there is no argument can be passed to the algorithms to disable GPU with the current version. I just made some adaptations on the tool to add the option to disable GPU for NDI and FANSI (which can be found in the GUI). Would you mind trying this and letting me know if it works for you? You can find the version in the 'no_gpu' branches of SEPIA:  https://github.com/kschan0214/sepia/tree/no_gpu

Kwok

**Reply by loneblues** [reply]
Hi. I tested the 'no_gpu' branch and now FANSI and NDI works without errors. 
I used the Enable GPU options, and got the error message again, however I cannot confirm that the processing is successful when you have a GPU, for obvious reasons. 
Thanks!

**Reply by kschan0214** [reply]
Thank you for testing! 

FYI: The 'no_gpu' branch is now merged to the master branch and tagged as 'v1.2.1.1'.
